packages/tools/devtools/devtools-core/src/data-visualization/DataEditing.ts

```diff
@@ -0,0 +1,43 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { ISharedObject } from "@fluidframework/shared-object-base";
+import { Serializable } from "@fluidframework/datastore-definitions";
+import { EditType } from "../CommonInterfaces";
+
+// Ideas:
```

Comment:
We should probably remove these TODO comments here. They don't really apply to editing.

Reply:
Check new commit